schedule/yast/sle/flows/default_sle15sp3_s390x.yaml

```diff
@@ -1,37 +1,45 @@
 ---
-name: qam-yast_self_update
-description: installation using self_update as boot parameter
-vars:
-  YUI_REST_API: 1
-schedule:
+# Default ordered sequence of steps for sle15sp3 with maintenance updates
```

Comment:
let's call it `_s390x_kvm.yaml` . Apply the same in the other file, for consistence with the same task we are doing for product validation.